shipctl/x86_64/Ubuntu_16.04/shippable_bump_version

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+shippable_bump_verison() {
+  local version_to_bump=$1
+  local action=$2
+  IFS='.' read -ra versionParts <<< "$version_to_bump"
+  IFS=' '
```

Comment:
This _might_ change the behaviour of the the shell if IFS was defined to be something else. We should probably keep a copy of the earlier IFS and restore after we read in our the version if we want to do this.

I would suggest using `cut` instead --

```
major=$(echo "1.3.4" | cut -d "." -f 1)
minor=$(echo "1.3.4" | cut -d "." -f 2)
patch=$(echo "1.3.4" | cut -d "." -f 3)
```